lib/tasks/after_tasks.rake

```diff
@@ -631,6 +631,25 @@ namespace :After do
     end
     puts && STDOUT.flush
   end
+
+  desc "Update each user's kudos with the user's id"
+  task(add_user_id_to_kudos: :environment) do
+    total_users = User.all.size
+    total_batches = (total_users + 999) / 1000
+    puts "Updating #{total_users} users' kudos in #{total_batches} batches"
+
+    User.find_in_batches.with_index do |batch, index|
```

Comment:
You can do `User.includes(:pseuds).find_in_batches` to batch pseud lookups and avoid n+1 queries.